Review Assignment/gulpFile.js

```diff
@@ -0,0 +1,18 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var imagemin = require('gulp-imagemin');
+var {series} = require('gulp');
+var concat = require('gulp-concat');
+
+exports.sass = function () {
+  return gulp.src('src/assets/stylesheets/sass/*.scss')
+    .pipe(sass())
+    .pipe(concat('main.css'))
+    .pipe(gulp.dest('src/assets/stylesheets/css'));
+}
+
+exports.imagemin = function(){
```

Comment:
function name should be camelcase seperated like imageMin. Please correct everywhere.